tests/compas/test_iotools.py

```diff
@@ -0,0 +1,125 @@
+import io
+import math
+import os
+import tempfile
+
+import pytest
+
+from compas import _iotools
+
+BASE_FOLDER = os.path.dirname(__file__)
+IMAGE_FILE_SIZE = 252391
+TEXT_FILE_SIZE = 747
+REMOTE_IMAGE_FILE_SIZE = 5430
+
+
+@pytest.fixture
+def path_image():
+    return os.path.join(BASE_FOLDER, 'fixtures', 'iotools', 'image.png')
+
+
+@pytest.fixture
+def path_text():
+    return os.path.join(BASE_FOLDER, 'fixtures', 'iotools', 'text.txt')
+
+
+@pytest.fixture
+def url_text():
+    return 'https://raw.githubusercontent.com/compas-dev/compas/master/README.md'
+
+
+@pytest.fixture
+def url_image():
+    return 'http://google.com/favicon.ico'
```

Comment:
Is this legal?  Maybe to be on the safe side choose a clearly open source image.